### PR TITLE
Journalist can add image

### PR DESCRIPTION
**Pull request by Ehugo2000**
[Journalist can add images to article ADMIN](https://www.pivotaltracker.com/story/show/176339915)
### User Story
```
As a journalist,
In order to make my articles more visually appealing, 
I would like to be able to upload and display images to my articles.
```
## Changes proposed in this pull request:
* Journalist can add image to article
* Journalist can see a preview
* Input fields will reset after creating an article
## What I have learned working on this feature:
* toBase64
* adding packages
* Semantic UI
## Packages/Gems added
* Cypress-file-upload
## Screenshots (Client)
Screenshot from 2021-01-14 15-45-49

### CODE REVIEW

File: src/modules/ArticlesServices.js [hunk 1]
```diff
@@ -13,12 +18,14 @@ const ArticlesServices = {
...
             body: event.target.body.value,
             category_id: parseInt(event.target.categories.value),
+            image: encodedImage,
           },
         },
         {
           headers: headers,
         }
       );
+      event.target.reset();
```
Comment (KyunginNa) [inline src/modules/ArticlesServices.js:28]: Is this for cleaning the inputs?

**Review by KyunginNa — COMMENTED** [review]
Looks good! Just left one question :)

File: src/modules/ArticlesServices.js (hunk 1 above)
Comment (Ehugo2000) [reply src/modules/ArticlesServices.js:28]: Yes! It will reset the input fields after creating an article.

**Review by emtalen — CHANGES_REQUESTED** [review]
It looks good but I want you to test for the article preview that you created. 
Also, personal opinion but it looks kind of weird that the lead is meta and greyed out and then also the category below but the body is not greyed in the middle (from the article preview in your screen shot), so have a look at that.

**Review by emtalen — APPROVED** [review]

**Pull request merged**